index.js

```diff
@@ -13,76 +17,115 @@
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
+const name = "marty"
```

Comment:
Great! The key here was using `const` because it doesn't allow us to reassign our variable